potpie/cli.py

```
@@ -0,0 +1,592 @@
+        typer.echo(f"Log file: {_log_file()}")
+        return
+    if _pid_file().exists():
+        _clear_pid()
+
+    _state_dir().mkdir(parents=True, exist_ok=True)
+    log_handle = _log_file().open("a")
+    log_handle.write(f"\n--- potpie start {datetime.now().isoformat()} ---\n")
+    log_handle.flush()
+
+    command = ["bash", str(script)]
+    process = subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)